src/Covid19Dashboard/WorldMapChart/index.jsx

```diff
@@ -125,6 +125,90 @@ class WorldMapChart extends React.Component {
       },
       hoverInfo: null,
     };
+    this.mapData = {
+      modeledCountyGeoJson: null,
+      densityGeoJson: null,
+      colors: {},
+      colorsAsList: null,
+      dotSizesAsList: null,
+    };
+  }
+  componentDidUpdate() {
+    if (!(this.mapData.colorsAsList === null
+      && Object.keys(this.props.jsonByLevel.county).length > 0)) {
+      return;
+    }
+    this.mapData.densityGeoJson = this.props.geoJson;
+    if (Object.keys(this.props.jsonByLevel.country).length && !this.choroCountryGeoJson) {
+      this.choroCountryGeoJson = addDataToGeoJsonBase(
+        this.props.jsonByLevel.country,
+        'country',
+      );
+      this.choroStateGeoJson = addDataToGeoJsonBase(
+        this.props.jsonByLevel.state,
+        'state',
+      );
+      this.choroCountyGeoJson = addDataToGeoJsonBase(
+        this.props.jsonByLevel.county,
+        'county',
+      );
+    }
+    this.mapData.modeledCountyGeoJson = filterCountyGeoJson(this.props.modeledFipsList);
+
+    // Finds max value in data set
+    const maxCountryVal = this.props.geoJson.features
+      .map(obj => obj.properties.confirmed)
+      .sort((a, b) => {
+        if (typeof a === 'string') {
+          return 1;
+        }
+        if (typeof b === 'string') {
+          return -1;
```

Comment:
is this because of "<5" strings? if so, please add a comment to explain what you're doing. And the IL map code should be able to handle this too